src/sagemaker/estimator.py

```diff
@@ -2147,6 +2147,11 @@ def __init__(
         self._hyperparameters = hyperparameters or {}
         self.checkpoint_s3_uri = checkpoint_s3_uri
         self.checkpoint_local_path = checkpoint_local_path
+
+        # Disable debugger if checkpointing is enabled by the customer
+        self.debugger_hook_config = \
```

Comment:
If a user specifies a custom hook config while checkpointing is enabled, should we log or print a message saying that debugger is disabled?

Reply:
Done